app/src/com/android/rkpdapp/provisioner/WidevineProvisioner.java

```diff
@@ -161,9 +162,15 @@ public Result provisionWidevine() {
 
     private byte[] fetchWidevineCertificate(MediaDrm.ProvisionRequest req) throws IOException {
         final byte[] data = req.getData();
+        String url = req.getDefaultUrl();
+	String wvHost = new URL(defaultUrl).getHost();
+        String override = WidevineProvisioningSettings.getServerUrlOverride(getApplicationContext());
+        if (override != null) {
+            url = url.replaceAll(wvHost, override);
```

Comment:
`replaceAll()` will lead to an incorrect result if `wvHost` string is included in other parts of the URL.

Need to construct a new URL based on default one, i.e.:
`new URL(defaultUrl.getProtocol(), /* host */ override, defaultUrl.getPort(), defaultUrl.getFile()).toString()`